docs/user_guide/faqs.md

```diff
@@ -0,0 +1,23 @@
+# Frequently Asked Questions
+
+### My entrypoint isn't working how I expect. How can I debug it?
```

Comment:
good call moving the debugging section out to faqs, tutorial flows a lot better too